Add additional medium types to mopeka (#34004)
home-assistant/home-assistant.github.io: source/_integrations/mopeka.markdown

The Mopeka integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.

> When setting up or changing options for a device, a 'Medium Type', the substance the sensor measures, such as Propane, can be chosen. The sensor needs to use the type of medium to adjust the coefficients used in the ultrasonic sensor calculations.

Reviewer: ```suggestion
When setting up or changing options for a device, a medium, the substance the sensor measures, such as propane, can be chosen. The sensor needs to use the type of medium to adjust the coefficients used in the ultrasonic sensor calculations.
```

## Supported Mediums

>  - Propane (default)
>  - Air (for unpressurized top-down sensors like water tanks)
>  - Fresh Water (for bottom-up sensors)
>  - Waste Water
>  - Live Well
>  - Black Water
>  - Raw Water
>  - Gasoline
>  - Diesel
>  - LNG
>  - Oil
>  - Hydraulic Oil

Reviewer: ```suggestion
- Propane (default)
- Air (for unpressurized top-down sensors like water tanks)
- Fresh Water (for bottom-up sensors)
- Waste Water
- Live Well
- Black Water
- Raw Water
- Gasoline
- Diesel
- LNG
- Oil
- Hydraulic Oil
```

The Mopeka integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.

> Once a device is discovered and selected for configuration, the user can choose a 'Medium Type' which is the substance the sensor meausres such as Propane.

Reviewer: ```suggestion
When setting up or changing options for a device, a 'Medium Type', the substance the sensor measures, such as Propane, can be chosen. The sensor needs to use the type of medium to adjust the coefficients used in the ultrasonic sensor calculations.
```

> The sensor needs to use the type of medium to adjust the coefficients used in the ultrasonic sensor calculations.

Reviewer: ```suggestion
```
